### PR TITLE
Contact_form

### PR DESCRIPTION
**Pull request by DhuruvSingh**
Added Contact Form in my portfolio

Kindly Review and give your valuable feedback

Thanks

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -208,6 +208,49 @@ <h2 class="skills"><span><img src="./Image/Rectangle-57.png" alt=""></span>Skill
...
         </div>
+        <div class="contact-form">
+            <h1>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+            <div class="form">
+                <form action="https://formspree.io/f/xeqnpraa" method="POST">
+                    <label><input type="text" name="name" maxlength="30" required=""
+                            placeholder="Full Name"></label>
+                    <label><input type="email" name="email" required="" placeholder="Email Address"></label>
+                    <label><textarea name="message" maxlength="500" cols="19" rows="10"
+                            placeholder="Enter Text Here" required=""></textarea></label>
```
Comment (zieeco) [inline index.html]: - [x] **OPTIONAL**
- kindly remove the `=""` from the `required attribute`. The required attribute is a boolean attribute that specifies that an input field must be filled out before submitting the form. It does not need any values added to it like other form attribute.

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @Team


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 
✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
_check the comment under the review_

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (DhuruvSingh) [reply index.html]: ok thanks

**Comment by DhuruvSingh** [comment]
Thank you for your valuable feedback @zieeco

**Pull request merged**